traveler_core_kit/src/main/java/com/guestlogix/travelercorekit/models/Traveler.java

```diff
@@ -34,21 +33,21 @@
      * @param apiKey             key provided by the Guestlogix platform. Invalid keys will result in forbidden response codes.
      * @param applicationContext application context where the sdk is running.
      */
-    public static void initialize(String apiKey, Context applicationContext) {
+    public static void initialize(String apiKey, Context applicationContext, boolean sandboxMode) {
```

Comment:
You're forcing the end users to use a sandbox mode boolean. Sandbox mode should be off by default. I think overloading the constructor here is will be better. Let me know what you think.

Reply:
updated